### PR TITLE
Update about page (id)

### PR DESCRIPTION
**Pull request by meisyal**
Comparing to about page in English, I saw some outdated translation on about page (Bahasa Indonesia). So, I decided to update it. @gozali help me to review the update.

Thanks :)

### CODE REVIEW

File: id/about/index.md [hunk 1]
```diff
@@ -169,27 +167,27 @@ digunakan pada setiap member instan `self.`
...
 
-* Ruby memiliki fitur-fitur yang menangani exception, seperti Java atau
-  Python, untuk mempermudah menangani error.
+* Ruby memiliki fitur-fitur yang menangani *exception*, seperti Java atau
+  Python, untuk mempermudah menangani *error*.
 
-* Ruby menyediakan mark-and-sweep garbage collector untuk semua obyek
-  Ruby. Tidak perlu me-maintain reference count pada library extension.
+* Ruby menyediakan *mark-and-sweep garbage collector* untuk semua objek
+  Ruby. Tidak perlu me-*maintain reference count* pada *extension*.
```
Comment (gozali) [inline id/about/index.md]: should it be menjaga or mempertahankan instead of me-maintain?

Comment (meisyal) [reply id/about/index.md]: I prefer "menjaga". What do you think?

Comment (gozali) [reply id/about/index.md]: Sounds good 👍

Comment (meisyal) [reply id/about/index.md]: I just updated it to "menjaga".

**Pull request merged**